src/excmds.ts

```diff
@@ -3089,48 +3088,63 @@ export function yank(...content: string[]) {
 }
 
 /**
- * Copies a string to the clipboard/selection buffer depending on the user's preferences
+ * Copies a string to the clipboard/selection buffer depending on the user's preferences.
  *
  * @hidden
  */
 //#background_helper
-async function setclip(str) {
-    // Functions to avoid retyping everything everywhere
+async function setclip(data: string) {
+    // Function to avoid retyping everything everywhere
+    const setclip_selection = () => Native.clipboard("set", data)
 
-    // Note: We're using fillcmdline here because exceptions are somehow not caught. We're rethrowing because otherwise the error message will be overwritten with the "yank successful" message.
-    const s = () => Native.clipboard("set", str)
-    const c = () => messageActiveTab("commandline_frame", "setClipboard", [str])
-
-    let promises = []
+    let promises: Array<Promise<any>> = []
     switch (await config.getAsync("yankto")) {
         case "selection":
-            promises = [s()]
+            promises = [setclip_selection()]
             break
         case "clipboard":
-            promises = [c()]
+            promises = [setclip_api(data)]
             break
         case "both":
-            promises = [s(), c()]
+            promises = [setclip_selection(), setclip_api(data)]
             break
     }
     return Promise.all(promises)
 }
 
+/**
+ * Copies a string to the clipboard using the Clipboard API.
+ * @hidden
+ */
+//#background_helper
+async function setclip_api(data: string) {
+    return window.navigator.clipboard.writeText(data)
+}
+
 /**
  * Fetches the content of the clipboard/selection buffer depending on user's preferences
  *
  * Exposed for use with [[composite]], e.g. `composite getclip | fillcmdline`
  */
 //#background
-export async function getclip(fromm?: "clipboard" | "selection") {
```

Comment:
`fromm` has been annoying me for years. I assumed it was because from was reserved. Has it just been a typo all these years?

Reply:
`from` *is* used as a keyword in import statements, but it [isn't reserved](https://www.ecma-international.org/ecma-262/#sec-keywords-and-reserved-words), probably because it can never be confused with an identifier. I've seen a few of these weirdly spelt names already – just today, I stumbled upon a `constructorr` in a context where `constructor` probably wouldn't have been reserved –, so it may just have been added by someone who *thought* it was reserved.